lib/api/objectPutACL.js

```diff
@@ -272,6 +274,9 @@ function objectPutACL(authInfo, request, log, cb) {
                             revisedAddACLParams);
                     });
             }
+            if (hasError) {
```

Comment:
same, should be called earlier : to avoid having a duplicate call via line 273...